packages/sdk/src/manifold/types.ts

```diff
@@ -0,0 +1,81 @@
+export enum ListingType {
+  INVALID,
+  INDIVIDUAL_AUCTION,
+  FIXED_PRICE,
+  DYNAMIC_PRICE,
+  RANKED_AUCTION,
+}
+
+export enum Spec {
+  NONE,
+  ERC721,
+  ERC1155,
+}
+
+export type BNHex = {
```

Comment:
This is just an `ethers.js` `BigNumber` represented as an object, should replace it with a regular `string`.